unit-tests/shared/src/test/scala/javalib/nio/file/UnixPathTest.scala

```diff
@@ -2,12 +2,22 @@ package javalib.nio.file
 
 import java.nio.file._
 
-import org.junit.Test
+import org.junit.{Test, Before}
 import org.junit.Assert._
+import org.junit.Assume._
 
 import scalanative.junit.utils.AssertThrows.assertThrows
+import org.scalanative.testsuite.utils.Platform.isWindows
 
-class PathTest {
+class UnixPathTest {
+
+  @Before
+  def checkIsUnix() {
```

Comment:
Same here: `assumeIsUnix`.

Also, perhaps this can be an `@BeforeClass` in the companion object instead.